libraries/lambda_handlers/handler.py

```diff
@@ -24,11 +24,12 @@ def handle(self, event, context):
         """
         self.logger.debug("EVENT:")
         self.logger.debug(json.dumps(event))
-        try:
+        # try:
+        if True:
             return self._handle(event, context)
-        except Exception as e:
-            self.logger.error(e.message, exc_info=1)
-            raise EnvironmentError('Bad Request: {}'.format(e.message))
+        # except Exception as e:
```

Comment:
@richmahn - if you remove exception trapping, what api caller get back if the request crashes the handler?  We want to be sure they don't get a crash dump.